Jenkinsfile

```diff
@@ -47,6 +47,9 @@ pipeline {
             }
         }
         stage('Build final version images') {
+            environment {
+                TAG_VERSION = readCurrentTag();
```

Comment:
I think that it is fine, but the current implementation of `readCurrectTag` has some issues.

Please, review this one: https://github.com/FromDoppler/hello-microservice/blob/c6151e852de2c1bd2600546339b5b0ab2d949a1a/Jenkinsfile#L105-L107

In https://stackoverflow.com/questions/56030364/buildingtag-always-returns-false you can get more context.

Reply:
Haha, in my implementation I am using `TAG_NAME` 😛

Are you sure that it is empty?

Reply:
You can check in the log in here http://docker.fromdoppler.com:8080/job/FromDoppler/job/doppler-webapp/job/master/ in the steps that use `TAG_NAME` 
![image](https://user-images.githubusercontent.com/2439363/124270692-ebfc3200-db12-11eb-8eee-50e72430b79d.png)

Reply:
And in the screen captures in the description of this PR I added the log for pipeline where you see the `TAG_NAME` is empty.
![image](https://user-images.githubusercontent.com/2439363/124271494-e7844900-db13-11eb-87d1-8864bdeb7683.png)

Reply:
En realidad, eso no debería ejecutarse, porque es master, no un tag.

Reply:
`master` no debería generar _final versions_, solo los _tags_ deben hacerlo.

Reply:
Es lo que te comentaba en el chat: https://makingsense.slack.com/archives/CPU4QDR6V/p1625226387003400?thread_ts=1625167475.001300&cid=CPU4QDR6V

Reply:
I've added the code as in hello microservice, the idea is test this to see if it works this way.